sdk/client-traits/src/lib.rs

```diff
@@ -7,25 +7,19 @@
 //! Asynchronous implementations are expected to create transactions, sign them, and send
 //! them but without waiting to see if the server accepted it.
 
-#![cfg(feature = "full")]
-
 use {
-    crate::{
-        clock::Slot,
-        commitment_config::CommitmentConfig,
-        epoch_info::EpochInfo,
-        hash::Hash,
-        instruction::Instruction,
-        message::Message,
-        pubkey::Pubkey,
-        signature::{Keypair, Signature},
-        signer::Signer,
-        signers::Signers,
-        system_instruction,
-        transaction::{self, Transaction, VersionedTransaction},
-        transport::Result,
-    },
     solana_account::Account,
+    solana_commitment_config::CommitmentConfig,
+    solana_epoch_info::EpochInfo,
+    solana_hash::Hash,
+    solana_instruction::Instruction,
+    solana_keypair::Keypair,
+    solana_program::{message::Message, system_instruction::transfer},
```

Comment:
Let's use `solana_message` and `solana_system_interface` to remove the `solana-program` dependency